packages/inter-protocol/src/auction/util.js

```diff
@@ -0,0 +1,30 @@
+import { M } from '@agoric/store';
+import {
+  multiplyRatios,
+  ratioGTE,
+} from '@agoric/zoe/src/contractSupport/index.js';
+
+export const DiscountOfferShape = M.any();
+export const PriceOfferShape = M.any();
```

Comment:
They were never used.

Reply:
Seem gone.

@turadg , I don't understand the `TODO_Shape` suggestion. Are you suggesting a new pattern type with this behavior?

Reply:
Yes, that's the idea put forth. We tend to use `any` as a placeholder for a forthcoming type, but that fails to signal that a shape is intended vs actually accepting any value.

I proposed a TODO shape instead of something like M.todo because this intent is orthogonal to the marcher lib. Also if it logged to console it would be the only shape that had side effects.